services/wfm/src/main/java/org/openkilda/wfm/topology/flow/service/FlowService.java

```diff
@@ -171,7 +171,6 @@ public FlowPair createFlow(Flow flow, String diverseFlowId, FlowCommandSender se
         try {
             result = (FlowPathsWithEncapsulation) getFailsafe().get(
                     () -> transactionManager.doInTransaction(() -> {
-                        ensureEncapsulationType(flow);
```

Comment:
not clear why have you removed that

Reply:
In previous patch I have added default encapsulation type to Flow model so we don't need to ensure it's not null manually now.